design-book/src/pane-layout.md

```diff
@@ -0,0 +1,21 @@
+# Pane Layout
+
+## Order of Importance
+
+- Generally, the most important fields/or the must used, should be at the top, quickly accessible. Less important widgets should be placed lower or in sub-panels
```

Comment:
```suggestion
- More important fields and tools should be given more visual prominence. This can be achieved with positioning, size, contrast or folding elements.
```